hamocc/mo_param_bgc.F90

```diff
@@ -520,6 +530,27 @@ module mo_param_bgc
 
 contains
 
+  !********************************************************************
+  subroutine ini_bgctimes(nday_in_year)
+
+    use mod_config, only: expcnf
+    ! NOTE: called also at run time after initialization
+    integer,intent(in) :: nday_in_year
+
+    days_per_year = real(nday_in_year)
+
+    if (nday_in_year /= 365 .and. mnproc==1 .and. lini .eqv. .true.) then
+      write (io_stdo_bgc,*) 'Error: Init iHAMOCC time variables: non-standard calendar selected with [days] ',days_per_year
```

Comment:
Sorry for being picky... but it would be nicer to write "Error: ..." only if it is treated as an error. Other wise it would be "Warning: ...". So maybe make a write statement for both cases using a if-then-else-block?

Reply:
Done.